From 0b481fd1753df058969a84dbd6bb3d05519222ca Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Tue, 11 Apr 2023 00:58:30 +0200
Subject: [PATCH 091/954] feat(stm32mp1-fdts): use iwdg1 instead of iwdg2

Change to use the iwdg1 by default on STM32MP1x boards.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 2aa622bc62db2c0a0807bc2a872912035a7abece feat(stm32mp1-fdts): use iwdg1 instead of iwdg2]
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: Ie37ecdc9af17fa21023fa0709d7b38f1dbd542a7
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/299851
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 fdts/stm32mp135f-dk.dts   | 2 +-
 fdts/stm32mp157a-ed1.dts  | 2 +-
 fdts/stm32mp157c-ed1.dts  | 2 +-
 fdts/stm32mp157d-ed1.dts  | 2 +-
 fdts/stm32mp157f-ed1.dts  | 2 +-
 fdts/stm32mp15xx-dkx.dtsi | 4 ++--
 6 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/fdts/stm32mp135f-dk.dts b/fdts/stm32mp135f-dk.dts
index b5c3db99a1..0028d27281 100644
--- a/fdts/stm32mp135f-dk.dts
+++ b/fdts/stm32mp135f-dk.dts
@@ -168,7 +168,7 @@
 	};
 };
 
-&iwdg2 {
+&iwdg1 {
 	timeout-sec = <32>;
 	status = "okay";
 };
diff --git a/fdts/stm32mp157a-ed1.dts b/fdts/stm32mp157a-ed1.dts
index 5c36f38783..04619c6f9b 100644
--- a/fdts/stm32mp157a-ed1.dts
+++ b/fdts/stm32mp157a-ed1.dts
@@ -175,7 +175,7 @@
 	};
 };
 
-&iwdg2 {
+&iwdg1 {
 	timeout-sec = <32>;
 	status = "okay";
 };
diff --git a/fdts/stm32mp157c-ed1.dts b/fdts/stm32mp157c-ed1.dts
index 013cf5ab0c..03bb242dfb 100644
--- a/fdts/stm32mp157c-ed1.dts
+++ b/fdts/stm32mp157c-ed1.dts
@@ -179,7 +179,7 @@
 	};
 };
 
-&iwdg2 {
+&iwdg1 {
 	timeout-sec = <32>;
 	status = "okay";
 };
diff --git a/fdts/stm32mp157d-ed1.dts b/fdts/stm32mp157d-ed1.dts
index 18d838732c..3c4cb0df49 100644
--- a/fdts/stm32mp157d-ed1.dts
+++ b/fdts/stm32mp157d-ed1.dts
@@ -175,7 +175,7 @@
 	};
 };
 
-&iwdg2 {
+&iwdg1 {
 	timeout-sec = <32>;
 	status = "okay";
 };
diff --git a/fdts/stm32mp157f-ed1.dts b/fdts/stm32mp157f-ed1.dts
index b5fe9a26d3..83ddc29fa2 100644
--- a/fdts/stm32mp157f-ed1.dts
+++ b/fdts/stm32mp157f-ed1.dts
@@ -179,7 +179,7 @@
 	};
 };
 
-&iwdg2 {
+&iwdg1 {
 	timeout-sec = <32>;
 	status = "okay";
 };
diff --git a/fdts/stm32mp15xx-dkx.dtsi b/fdts/stm32mp15xx-dkx.dtsi
index a3097907f9..7cbdcb9734 100644
--- a/fdts/stm32mp15xx-dkx.dtsi
+++ b/fdts/stm32mp15xx-dkx.dtsi
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: (GPL-2.0+ OR BSD-3-Clause)
 /*
- * Copyright (c) 2019-2022, STMicroelectronics - All Rights Reserved
+ * Copyright (c) 2019-2023, STMicroelectronics - All Rights Reserved
  * Author: Alexandre Torgue <alexandre.torgue@st.com> for STMicroelectronics.
  */
 
@@ -183,7 +183,7 @@
 	};
 };
 
-&iwdg2 {
+&iwdg1 {
 	timeout-sec = <32>;
 	status = "okay";
 };
-- 
2.53.0

